src/freenas/usr/local/libexec/nas/generate_sssd_conf.py

```diff
@@ -741,7 +741,7 @@ def add_activedirectory_section(client, sc):
     ad = client.call('notifier.directoryservice', 'AD')
```

Comment:
Perhaps Struct(client.call('notifier.directoryservice', 'AD')) only would suffice?

Reply:
Yes it is sufficient.